### ISSUE TITLE
Upload contacts (super admin)

### ISSUE THREAD

**Issue by abhi1203** [glific/glific-frontend#1668]
**Is your feature request related to a problem? Please describe.**
[Requirements](https://docs.google.com/document/d/1Q7ozYAHl1wcVQlXcg3iZjQZPHaUQJzQCYHTLaajncrs/edit#bookmark=id.ma2axckq07dg)

**Describe the solution you'd like**
[Design flow](https://www.sketch.com/s/ba323b85-11d7-4687-8a19-681a32132bab/p/2E8117A3-0AEC-4318-9330-2C1546FE59DF)

**Comment by kurund** [comment]
@mdshamoon 

The design looks good to me. We can start the implementation.

**Comment by kurund** [comment]
@abhi1203 

Few thoughts:

* This might be a good functionality for NGO admin too. Any thoughts on that?
* I feel we should move import as an action item in the organization listing. This will eliminate the need to choose the organization in the upload screen and reduce potential user errors.
* As we are allowing bulk upload now, we can also want to think about bulk delete action at some point.

**Comment by abhi1203** [comment]
@kurund yes, it will be made available to NGO admin in future based on the requirement and reception.
Sure, i didn't consider combining it with the org listing page because i thought it might be easier for FE team to make it available for NGO admin if it was a separate piece. If you think we can still do that, im okay with moving it to org page.
Also from your exp. doo you think we should track who we're uploading contacts for and the meta information.

Bulk delete also makes sense. Sure, we can keep an eye on if an NGO ever requests for it.

**Comment by kurund** [comment]
@abhi1203 

If we have plans to make it available to NGO admin then it makes sense to have a separate page. We will only move the upload action to row-level than action on top to reduce clicks and errors.

**Comment by abhi1203** [comment]
@kurund  cool so you'll list all the orgs beforehand on that page? That might be okay too. So like we add consulting hours.

**Comment by kurund** [comment]
@abhi1203 

We will have upload icon before delete icon on this page: https://www.sketch.com/s/ba323b85-11d7-4687-8a19-681a32132bab/a/YG1z9rO

**Comment by abhi1203** [comment]
@kurund @mdshamoon I've updated this. let me know if it works.
